drivers/gamatronic.c

```diff
@@ -358,11 +359,12 @@ void setup_serial(const char *port)
     if (i == 5) {
 	printf("Can't talk to UPS on port %s!\n",port);
 	printf("Check the cabling and portname and try again\n");
-	printf("Please note that this driver only support UPS Models with SEC Protorol\n");
+	printf("Please note that this driver only support UPS Models with SEC Protocol\n");
```

Comment:
Indentation issues (spaces vs tabs)?

And if we `exit()` in this clause, the `else` below seems superfluous (is it for `cppcheck` to get quiet?)